.github/workflows/tsfc-tests.yml

```diff
@@ -28,18 +28,18 @@ jobs:
           pip3 install .
 
       - name: Clone tsfc
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           path: ./tsfc
           repository: firedrakeproject/tsfc
-          ref: master
```

Comment:
@mscroggs How do we revert this change?

Reply:
If we merge https://github.com/FEniCS/ufl/pull/326 then update in this fork that should do it I hope